File: Exercício_13.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
 float base, altura,area;
 printf("Digite o valor da base:");
 scanf("%f",&base);

 printf("Digite o valor da altura:");
 scanf("%f",&altura);
 area=base*altura;
 printf("Terreno ");
    if(area>100)
    {
      printf("Grande");
    }
    else
    {
       printf("Pequeno");
    }
 return 0;
}

File: Exercício_13.py
base = float(input("Digite o valor da base: "))
altura = float(input("Digite o valor da altura: "))

area = base * altura

print(f"o valor da area do terreno e de : {area:.2f} ")

if(area > 100):
    print(f"terreno grande")
else:
    print(f"terreno pequeno")